themes-default/slim/views/addShows_newShow.mako

```diff
@@ -215,19 +207,67 @@ const startVue = () => {
             }
         },
         methods: {
-            vueSubmitForm,
-            submitForm() {
-                // If they haven't picked a show or a root dir don't let them submit
-                if (this.addButtonDisabled) {
-                    this.$snotify.warning('You must choose a show and a parent folder to continue.');
-                    return;
+            async submitForm(skipShow) {
+                const { currentSearch, addButtonDisabled } = this;
+
+                let formData;
+
+                if (skipShow && skipShow === true) {
+                    formData = new FormData();
+                    formData.append('skipShow', 'true');
+
+                    if (currentSearch.cancel) {
+                        // Abort current search
+                        currentSearch.cancel();
+                        currentSearch.cancel = null;
+                    }
+                } else {
+                    // If they haven't picked a show or a root dir don't let them submit
+                    if (addButtonDisabled) {
+                        this.$snotify.warning('You must choose a show and a parent folder to continue.');
+                        return;
+                    }
+
+                    // Converts select boxes to command separated values [js/blackwhite.js]
+                    generateBlackWhiteList(); // eslint-disable-line no-undef
+
+                    formData = new FormData(this.$refs.addShowForm);
+                }
+
+                this.otherShows.forEach(nextShow => formData.append('other_shows', nextShow));
+
+                const response = await apiRoute.post('addShows/addNewShow', formData);
+                const { data } = response;
+                const { result, message, redirect, params } = data;
+
+                if (message.length !== 0) {
```

Comment:
Since this isn't an array use `if (message) {}`. Same goes for the `redirect.length` below.